src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/DocLint.java

```diff
@@ -0,0 +1,446 @@
+/*
+ * Copyright (c) 2012, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright year should be updated

Reply:
I'll try and get into a new habit of updating copyrights earlier. Personally, I think these updates are unnecessary noise in a review, and I would usually update files after the review but before pushing the changes. I guess that is not so easy anymore, since I don't want to affect "Reviewed" status by committing copyright year changes.